Automattic/sensei
# Fetch data from SenseiLMS.com for Sensei Home

Fixes #5731 
Fixes #5740 

### Changes proposed in this Pull Request

* Adds new helper for Sensei Home data from SenseiLMS.com.
* Wires up providers for news and guides.
* This helper will also feed required data for local notices and extensions.
* Note: For now I'm not wiring up the extensions. We might need to transform the data coming from SenseiLMS.com.

### Testing instructions
Test the REST API endpoint using the method you prefer. If you use the basic auth plugin, you can do something like this:
```
curl -s -X GET --user jake:jakejake 'http://sensei.local/wp-json/sensei-internal/v1/home' | jq '.news,.guides'
```

Verify the news and guides show up correctly. 


### New/Updated Hooks

* `sensei_home_remote_data_primary_plugin_slug`: Allows Sensei Pro to assert itself as the primary plugin.
* `sensei_home_remote_data_other_plugins`: Allows us to one day allow Sensei Block content to show up on Home when Sensei LMS is installed.
